orchagent/routeorch.cpp

```diff
@@ -466,6 +466,21 @@ void RouteOrch::doTask(Consumer& consumer)
             vector<string> ipv = tokenize(ips, ',');
             vector<string> alsv = tokenize(aliases, ',');
 
+            /*
+             * To check the validity of nexthop and make sure it works even in
+             * abnormal cases. One case is old format - empty "nexthop",
+             * then set ip to zero for backward compatibility.
+             */
+            if (alsv.size() == 0)
+            {
+                it = consumer.m_toSync.erase(it);
+                continue;
+            }
+            else if (alsv.size() != ipv.size())
+            {
+                ipv.resize(alsv.size(), "0.0.0.0");
```

Comment:
How about IPv6? Since `nexthop `is a set of entries, will there be a case where we can have a mix of valid ips and empty strings `""`?